eahub/config/urls.py

```diff
@@ -21,6 +21,7 @@
     path('groups/', views.groups, name='groups'),
     path('admin/', admin.site.urls, name='admin'),
     path('about/', views.about, name='about'),
+    path('newsletter/', TemplateView.as_view(template_name='eahub/newsletter.html'), name='newsletter'),
```

Comment:
We shouldn't have the same content at two different URLs (and we _can't_ have two URL patterns in Django with the same name; that will break things). One of these should redirect to the other.

Reply:
Did you mean for one of these to be a redirect?

Reply:
Yeah, I made a dumb guess. I've read In here on how to do this properly (I think): https://stackoverflow.com/questions/42614172/how-can-i-redirect-from-view-in-django/42614265, but I guess I'll hold on until I have a local environment running.

Reply:
Use the `LegacyRedirectView`; we've got a bunch of these elsewhere in the file.

Reply:
Lol, you're right. Done that.